changelog/unreleased/features/6886.md

```diff
@@ -0,0 +1,3 @@
+- Added custom header x-mapbox-app-info to requests.
```

Comment:
Is this relevant for developers? I think we can skip it.

Reply:
Wasn't this what our customer requested? I mean in the `change-management/issues/1222` "customer impact" is filled.

Reply:
My understanding is that this is an internal feature, customers will not need to interact with these headers directly.

Reply:
Removed this entry.